### PR TITLE
Prepare for v2.3.0

### PR DESCRIPTION
**Pull request by vstehle**
Update the changelog and provision the conformance profile GUID for v2.3.0.

### CODE REVIEW

**Comment by xypron** [comment]
```
$ git shortlog v2.2.0..HEAD
Vincent Stehlé (15):
      CI: use apt-get to fix warnings
      trivial: move some footnotes closer to their referrer
      chapter2: disambiguate collation protocol
      chapter3: deprecate spin tables
      chapter2: require authenticated fmp capsules for fw update
      trivial: add one typo to our list
      references: bump psci version
      references: bump dt-schema version
      references: bump bbr version
      Update UEFI version to 2.10 A
      CI: specify Sphinx version to fix the build
      CI: specify ubuntu version to fix the build
      references: bump devicetree schema tools, psci and smccc
      trivial: add required global variables anchor
      chapter2: add boot manager requirements
```
Deprecating spin tables seem to be missing in the changelog.

Otherwise
Reviewed-by: @xypron

**Comment by vstehle** [comment]
Hi @xypron, thanks for the review (and @ricardosalveti). You made me doubt; but yes, spin table deprecation is there.

File: source/chapter2-uefi.rst [hunk 1]
```diff
@@ -360,6 +360,14 @@ specification [#VersionsNote]_.
     { 0x9073eed4, 0xe50d, 0x11ee, \
     { 0xb8, 0xb0, 0x8b, 0x68, 0xda, 0x62, 0xfc, 0x80 }}
 
+- Version 2.3.x:
+
+.. code-block:: c
+
+    #define EFI_CONFORMANCE_PROFILE_EBBR_2_3_GUID \
+    {0x7721fc77, 0xa724, 0x11ef, \
+    {0x8e, 0xaa, 0xf7, 0xc9, 0xb1, 0x94, 0xba, 0x75}}
```
[the remark this answers is not in the document]
Comment (vstehle) [reply source/chapter2-uefi.rst]: Thanks @etienne-lms; this is fixed now.

**Comment by vstehle** [comment]
Approved during [the EBBR call of Dec 4](https://github.com/ARM-software/ebbr/wiki/EBBR-Notes-2024.12.04); we should release EBBR v2.3.0 soon.

**Pull request merged**